cabot/metricsapp/models/grafana.py

```diff
@@ -90,3 +90,4 @@ class Meta:
     panel_id = models.IntegerField()
     series_ids = models.CharField(max_length=20)
     selected_series = models.CharField(max_length=20)
+    panel_url = models.CharField(max_length=400, null=True)
```

Comment:
Some of the longer ones we have that I found were around 350...I can increase it for more of a buffer. 600?